src/parser/warning.mli

```diff
@@ -0,0 +1,8 @@
+type t = { location : Location.span; message : string }
```

Comment:
The way this file has been moved with changes loses its git history and easy access to its authorship information.